ibm_catalog.json

```diff
@@ -9,14 +9,14 @@
         "target_terraform",
         "terraform",
         "solution",
-        "containers",
-        "converged_infra"
```

Comment:
Removing this is going to be a problem because Partner Centre is going to add it back and add it to the top of the category list - please add it back